2403 document featddl allow drop generated column in table with schema registry (#2405)
risingwavelabs/risingwave-docs: docs/sql/commands/sql-alter-table.md

The `ALTER TABLE` command modifies the definition of a table.

> :::note
>  If your table is defined with a schema registry, for the following clauses, you can typically only use the `REFRESH SCHEMA` clause to alter it, with an exception for [tables with generated columns](#drop-column).
> :::

Reviewer: ```suggestion
```

:::note

> + If your table is defined with a schema registry, its column can not be altered. However, if the table with a schema registry has generated columns, you can drop the generated columns, and note dropping these generated columns will trigger refreshing schema.

Reviewer 2: ```suggestion
+ If your table is defined with a schema registry, its column can not be altered. However, if the table with a schema registry has generated columns, you can drop the generated columns. Note that dropping these generated columns will trigger a schema refresh.
```

> + If your table is defined with a schema registry, its column can not be altered. However, if the table with a schema registry has generated columns, you can drop the generated columns. Note that dropping these generated columns will trigger a schema refresh.

Reviewer 3: ```suggestion
+ If your table is defined with a schema registry, you can only change the table schema by `ALTER TABLE t REFRESH SCHEMA`. One exception is you can drop the generated columns even if the schema is defined with a schema registry. Note that dropping these generated columns will trigger a schema refresh.
```

:::note

> + As mentioned at the beginning of this page, for tables with a schema registry, you can generally only use the `REFRESH SCHEMA` clause. However, there's an exception for tables with generated columns. You can drop these generated columns even if the schema is defined in a registry. Note that dropping these generated columns will trigger a schema refresh.

Reviewer: ```suggestion
+ If your table is defined with a schema registry, you can only change the table schema by `ALTER TABLE t REFRESH SCHEMA`. One exception is you can drop the generated columns even if the schema is defined with a schema registry. Note that dropping these generated columns will trigger a schema refresh.
```
